spec/services/hyrax/import_url_failure_service_spec.rb

```diff
@@ -5,14 +5,15 @@
   let(:file) do
     create_for_repository(:file_set, user: depositor)
   end
+  let(:change_set) { Hyrax::FileSetChangeSet.new(file) }
 
   before do
-    allow(file.errors).to receive(:full_messages).and_return(['huge mistake'])
+    allow(change_set.errors).to receive(:full_messages).and_return(['huge mistake'])
   end
 
   describe "#call" do
     before do
-      described_class.new(file, depositor).call
+      described_class.new(change_set, depositor).call
```

Comment:
You are calling `ImportUrlFailureService` with a change set, but in `app/` it's called with a `FileSet`

Reply:
@jcoyne - do the full error messages need to be in the email? If they are no longer in the file set, I guess I'm not sure where this service has access to them.